src/main.cpp

```diff
@@ -1100,6 +1100,20 @@ int main(int Argc, char *Argv[]) {
         exit(1);
     }
 
+    if (strcmp(file, "-")) {
```

Comment:
`strncmp`, otherwise it will not pass security review.

Reply:
fixed